.gitignore

```diff
@@ -3,4 +3,6 @@ dist
 node_modules
 .vscode-test*
 *.vsix
-*.node
\ No newline at end of file
+*.node
+*.qps-ploc.json
+l10n/bundle.l10n.json
```

Comment:
Couple things. 
- Why do we ignore l10/bundle.l10n.json? 
- Will `.qps-ploc.json` _need_ to be ignored (i.e. will it be generated on the main branch at all, or is it just something you had setup for testing?)

Reply:
Those files are auto-generated by the `pseudoLocalize` script by static scanning the source code. So they don't need to be committed. Just generate them with the source code you need.
The pseudo file will be generated when you test pseudo locale each time so *need* to be ignored.

Reply:
So I guess what I don't understand is how will we commit translations to the repository. Is there a point where we'll have things in `l10n/` like, `bundle.l10n.zh-cn.json` (or something similar)?

Reply:
Yes, real translations need to be committed, like `l10n/bundle.l10n.zh-cn.json`. They won't be ignored.